nifty-examples/src/test/java/com/facebook/nifty/server/TestNiftySSLServer.java

```diff
@@ -220,6 +224,42 @@ public void testSSLSessionResumption() throws Exception {
         Assert.assertTrue(isSessionResumed(getSSLSession(client4)));
     }
 
+    class TestConfigUpdater implements SSLConfigUpdater {
```

Comment:
It looks like this interface only has the `attachTransport` and `detachTransport` methods. Should it be called more generically, like `TransportAttachObserver`, with a `ConfigUpdater` being just one use case of that?